packages/react/src/banner/Banner.tsx

```diff
@@ -31,28 +33,38 @@ export const Banner = forwardRef<HTMLDivElement, BannerProps>(
     { children, className, colorScheme = "neutral", onClose, ...props },
     ref,
   ) => {
+    const descriptionId = useId();
+    const labelId = useId();
+
     return (
-      <Flex ref={ref} {...styles.banner({ colorScheme }, className)} {...props}>
-        <Icon asChild {...styles.icon({ colorScheme })}>
-          {createElement(iconMap[colorScheme])}
-        </Icon>
+      <BannerContextProvider descriptionId={descriptionId} labelId={labelId}>
+        <Flex
+          ref={ref}
+          role="alert"
```

Comment:
Is alert the correct role for it vai? or we can use `role="banner"` here?

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/banner_role

Reply:
no - banner is basically a replacement for the `<header>` element - and is more suited for a global header/topbar or `Skip to content` link

Reply:
Got it vai.